### ISSUE TITLE
ParseError when running broserify task

### ISSUE THREAD

**Issue by reckoner165** [mmathias01/google-charts#14]
I added an import command in my project and snippet from your simple example code:
`import { GoogleCharts } from 'google-charts';`

However, the browserify task throws the following error: 
```
>> /Users/[...]/node_modules/google-charts/googleCharts.js:44
>> export let GoogleCharts = new googleCharts();
>> ^
>> ParseError: 'import' and 'export' may appear only with 'sourceType: module'
```
I use the `es2015` babel preset in browserify Although changing this to `env` gives the same error.
Am I missing something?

**Comment by QuentinMcCarthy** [comment]
Seems like a similar problem to #15 in that the code should be module.exports instead of simple export.

Pull request for the fix at #17

**Comment by mmathias01** [comment]
the import is an ES6 module which works best for tree shaking and module bundling but which doesn't work so well for some situations. I updated the build to be CJS and MJS so hopefully this issue is resolved now.
